src/BizHawk.Client.Common/lua/LuaHelper.cs

```diff
@@ -9,5 +10,37 @@ public static class LuaExtensions
 	{
 		public static LuaTable EnumerateToLuaTable<T>(this NLuaTableHelper tableHelper, IEnumerable<T> list, int indexFrom = 1)
 			=> tableHelper.ListToTable(list as IReadOnlyList<T> ?? list.ToList(), indexFrom);
+
+		public static string Serialize(this LuaTable lti)

```

Comment:
Maybe call it `PrettyPrintShallow` or something to indicate it's not for general use